### DISCUSSION TITLE
Documentation for --speaker_idx etc.

### DISCUSSION THREAD

**Discussion by 54696d21** [coqui-ai/TTS#429] [General]
Is there any documentation or other reading hints for these parameters?

```
--gst_style, --speaker_idx, --speakers_json
```

**Comment by erogol** [comment]
if you mean with the ```tts``` command, multi-speaker models are not supported yet. And there is no multi-speaker model released to be used with the ```tts``` command.

But you can try this notebook https://colab.research.google.com/drive/1-xI9HiG5B-APnwe7KQwtOBPp1gggg-jZ?usp=sharing

**Reply by 54696d21** [reply]
and are GSTs supported? and is there an example for it?

**Reply by erogol** [reply]
All are supported for training but GST is not also supported for inference with ```tts```.
